packages/sitecore-jss-react/src/components/Placeholder.tsx

```diff
@@ -71,10 +73,19 @@ class PlaceholderComponent extends PlaceholderCommon<PlaceholderComponentProps>
   }
 
   render() {
+    const {
+      name,
+      rendering: { placeholders },
+    } = this.props;
     const childProps: PlaceholderComponentProps = { ...this.props };
 
     delete childProps.componentFactory;
 
+    if (placeholders && placeholders[PREFIX_PLACEHOLDER]) {
```

Comment:
Agree, it seems like this logic could be done in `PlaceholderCommon.getPlaceholderDataFromRenderingData`. Is this not possible for some reason? 

For comments, is it really a "TODO"? Seems like this is all set. Also, might be good to include "SXA" in the comment so that it's clear this is related to SXA.